docs/content/v2.14/reference/configuration/yb-tserver.md

```diff
@@ -388,6 +388,12 @@ Interval at which the tablet manager tries to cleanup split tablets that are no
 
 Default: `60`
 
+##### --enable_automatic_tablet_splitting
+
+Enables YugabyteDB to [automatically split tablets](../../../architecture/docdb-sharding/tablet-splitting/#automatic-tablet-splitting), based on the specified tablet threshold sizes configured below.
```

Comment:
No threshold sizes are specified for yb-tserver, I think this should be changed to
either
```
Enables YugabyteDB to [automatically split tablets](../../../architecture/docdb-sharding/tablet-splitting/#automatic-tablet-splitting), based on the specified tablet threshold sizes.
```
or even shorter (because no sizes are configured at yb-tserver)
```
Enables YugabyteDB to [automatically split tablets](../../../architecture/docdb-sharding/tablet-splitting/#automatic-tablet-splitting).
```
This should be applied for every `yb-tserver.md` for 2.14, 2.16, stable and preview versions.